momepy/utils.py

```diff
@@ -470,20 +470,19 @@ def limit_range(vals, rng):
     array
         limited array
     """
+
     vals = np.asarray(vals)
-    if len(vals) > 2:
+
+    if len(vals) > 2 or not np.isnan(vals).all():
```

Comment:
Didn't you mean `and` instead of `or`?

Reply:
Yes, I think my brain is a bit foggy.

As a side note, why not `len(vals) > 1`?